index.js

```diff
@@ -145,6 +146,7 @@ async function httpProxy (fastify, opts) {
   fromOpts.rewriteHeaders = rewriteHeaders
 
   fastify.register(From, fromOpts)
+  fastify.register(UrlData)
```

Comment:
This is a good idea, but we should fix this too:
https://github.com/fastify/fastify-url-data/issues/88

A user that has already this plugin would get a startup error

Reply:
That's good point.

Reply:
It is way better to check if `fastify-url-data` registered inside this project, instead of suppressing the error from `fastify-url-data` side.